source/configuration_manual/howto/director_with_lua.rst

```diff
@@ -0,0 +1,109 @@
+.. _howto_director_lua:
+
+==============================
+Run director with Lua on proxy
+==============================
+
+Small scale deployments using NFS or other shared storage still need some way to ensure users
+end up in same backend. For this purpose, a simple Lua script can be used.
+
+The script can be ran on a proxy directly, or one can have one dedicated proxy node acting
+as director node running the script.
+
+Prerequisites
+-------------
+
+To run the script, you need to configure a database on one of the supported databases.
```

Comment:
Maybe: "To run this example script, you need to setup tracking using one of Dovecot's supported database backends."